Dockerfile

```diff
@@ -28,4 +28,4 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
 
 RUN ln -sf /usr/share/zoneinfo/America/New_York /etc/localtime
 
-CMD ["gunicorn", "packet:app", "--bind=0.0.0.0:8080", "--access-logfile=-"]
+CMD ["gunicorn", "packet:app", "--bind=0.0.0.0:8080", "--access-logfile=-", "--timeout=600"]
```

Comment:
When building the initial cache, sometimes ldap can take a while which causes gunicorn to crash. This prevents this from occurring during the initial cache creation and prevents packet from return 504s